### PR TITLE
User templatetags

### PR DESCRIPTION
**Pull request by phlax**
improves/updates profile template inclusion tags, hides permission info to anon, and some other cleanups

### CODE REVIEW

File: pootle/apps/pootle_profile/templatetags/profile_tags.py [hunk 1]
```diff
@@ -87,17 +88,27 @@ def profile_social(profile):
...
         info["url"] = reverse(
             "pootle-language-browse",
             kwargs=dict(language_code=code))
+    if teams:
+        teams_title = _(
+            "%s's language teams"
+            % profile.user.display_name)
+    else:
+        teams_title = _(
+            "%s is not a member of any language teams"
```
Comment (unho) [inline pootle/apps/pootle_profile/templatetags/profile_tags.py]: teams? Shouldn't be "team"?

File: pootle/apps/pootle_profile/templatetags/profile_tags.py [hunk 2]
```diff
@@ -87,17 +88,27 @@ def profile_social(profile):
 
 
 @register.inclusion_tag("user/includes/profile_teams.html")
-def profile_teams(profile):
+def profile_teams(request, profile):
```
Comment (unho) [inline pootle/apps/pootle_profile/templatetags/profile_tags.py]: Is `request` actually necessary?

File: pootle/apps/pootle_profile/templatetags/profile_tags.py (hunk 1 above)
Comment (phlax) [inline pootle/apps/pootle_profile/templatetags/profile_tags.py]: its the teams list

File: pootle/apps/pootle_profile/templatetags/profile_tags.py (hunk 2 above)
Comment (phlax) [inline pootle/apps/pootle_profile/templatetags/profile_tags.py]: yes, in order to restrict what is viewable - that or permissions, but for now it only restricts anon so the request.user is kinda needed

**Comment by unho** [comment]
lgtm

**Pull request merged**